client/dashboard/sites/overview/site-action-menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from '@tanstack/react-query';
+import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { moreVertical } from '@wordpress/icons';
+import { addQueryArgs } from '@wordpress/url';
+import { useAnalytics } from '../../app/analytics';
+import { isFSEActiveQuery } from '../../app/queries/site-themes';
+import { getSiteEditUrl } from '../../utils/site-url';
+import type { Site } from '../../data/types';
+
+const SiteActionMenu = ( { site }: { site: Site } ) => {
+	const { recordTracksEvent } = useAnalytics();
+	const { data: isFSEActive, isLoading: isFSEActiveLoading } = useQuery(
+		isFSEActiveQuery( site.ID )
+	);
+
+	const trackActionClick = ( action: string ) => {
+		recordTracksEvent( 'calypso_sites_dashboard_site_action_menu_click', { action } );
+	};
+
+	const handleEditSite = () => {
+		trackActionClick( 'edit_site' );
+		window.open( getSiteEditUrl( site, isFSEActive ), '_blank' );
+	};
+
+	const handleWritePost = () => {
+		trackActionClick( 'write_post' );
+		window.open( `${ site.options?.admin_url }post-new.php`, '_blank' );
+	};
+
+	const handleImportSite = () => {
+		trackActionClick( 'import_site' );
+		window.open( addQueryArgs( '/setup/site-migration', { siteSlug: site.slug } ), '_blank' );
+	};
+
+	return (
+		<DropdownMenu icon={ moreVertical } label={ __( 'Quick actions' ) }>
+			{ () => (
+				<MenuGroup>
+					<MenuItem disabled={ isFSEActiveLoading } onClick={ handleEditSite }>
+						{ __( 'Edit site ↗' ) }
+					</MenuItem>
+					<MenuItem onClick={ handleWritePost }>{ __( 'Write a post ↗' ) }</MenuItem>
+					<MenuItem onClick={ handleImportSite }>{ __( 'Import site ↗' ) }</MenuItem>
```

Comment:
I am confused with this action. Is it a very common action to import a site into another live site, that it deserves this location? Particularly if I already have a paid plan and a working site etc... cc: @matt-west

Reply:
I'd argue it's specially important for non-wpcom sites since we want to migrate them to WP.com.
<img width="536" height="634" alt="SCR-20250722-pivb" src="https://github.com/user-attachments/assets/35bb707a-c65a-43ea-b3d2-f4e619155c41" />

That said, there's also an existing logic in v1 Site List where we take users to wordpress.com/move

https://github.com/Automattic/wp-calypso/blob/de78d73f8368a56ccb8eb4314bc95c0054846b8e/client/sites/components/sites-dataviews/actions.tsx#L228-L235

@matt-west any preference?

Reply:
I think it can be both. For WP.com sites you are importing into the existing site, for non-WP.com sites you are importing to WordPress.com.<br><br>I’m not sure if the /move flow currently supports that though. We might need to send users with an existing site to the import page within wp admin instead.